XRTK-Core/Packages/com.xrtk.core/Inspectors/Utilities/Packages/MixedRealityPackageUtilities.cs

```diff
@@ -209,7 +209,8 @@ internal static async void ValidatePackages()
 
         private static async Task AddPackageAsync(MixedRealityPackageInfo packageInfo)
         {
-            var tag = (await GitUtilities.GetAllTagsFromRemoteAsync(packageInfo.Uri)).LastOrDefault();
+            var versionSeperator = new char[] { '.' };
```

Comment:
This could be a readonly field in the class so we don't allocate it every time we add a package.